src/tools.rs

```diff
@@ -232,7 +232,7 @@ pub fn guess_language<P: AsRef<Path>>(buf: &[u8], path: P) -> (Option<LANG>, Str
 pub(crate) fn remove_blank_lines(data: &mut Vec<u8>) {
     let count_trailing = data.iter().rev().take_while(|&c| *c == b'\n').count();
     if count_trailing > 0 {
-        data.truncate(data.len() - count_trailing + 1);
```

Comment:
Why is this change required?

Reply:
During my tests I noticed that few DeepSpeech source files generated different json files in Windows and Linux. Thanks to @Luni-4 and @lu-zero help we determined that the cause of those differences were the end of line formats at the end of few downloaded source files. This line should fix this issue and allow the generation of the same json files both in Windows and in Linux.

Reply:
Unless I'm mistaken, after your change this line is basically doing nothing (truncating `data` to its full length) and the `remove_blank_lines` function wouldn't actually remove blank lines.
@calixteman why was `remove_blank_lines` necessary?

Reply:
I added that function to remove the blank lines at the end of a file, such that we don't consider them for the `Blank` metric counting. But on Windows new lines are identified by `/r/n`, so we have a different `data` length when we compare the source files parsed on Linux with the ones parsed on Windows. One solution could be leaving blank lines at the end of a file, as implemented in this PR (I agree with this approach because blank lines at the end of a file should be considered), otherwise we can find `\r\n` on Windows

Reply:
> One solution could be leaving blank lines at the end of a file, as implemented in this PR

So we could just remove the `remove_blank_lines` function entirely?

Reply:
Looking more carefully the code, `remove_blank_lines` removes blank lines when `count_trailing > 0` leaving a single new line at the end of a file. So my first solution is not feasible, the second one is correct instead